### ISSUE TITLE
Remove the unnecessary SecMarker in all rules #REQUEST-941-APPLICATION-ATTACK-XSS

### ISSUE THREAD

**Issue by umarfarook882** [SpiderLabs/owasp-modsecurity-crs#818]
In some of the rules,  unnecessary SecMarker are there, so please check it and remove it, if not needed in upcoming release :)

`REQUEST-941-APPLICATION-ATTACK-XSS.conf: line no:739 SecMarker END_XSS_CHECKS`

**Comment by dune73** [comment]
Thanks for reporting @umarfarook882.

There are quite a lot of other markers, but I think all the other ones are actively used in `skipAfter` actions.

Here is me take on the number of references for the existing markers in the rules:

```
END-SAMPLING : 1
END-DRUPAL-RULE-EXCLUSIONS : 1
END-WORDPRESS-ADMIN : 2
END-WORDPRESS : 4
END_RBL_LOOKUP : 1
END_RBL_CHECK : 5
END-REQUEST-910-IP-REPUTATION : 8
END-REQUEST-911-METHOD-ENFORCEMENT : 8
END-REQUEST-912-DOS-PROTECTION : 9
END_DOS_PROTECTION_CHECKS : 3
END-REQUEST-913-SCANNER-DETECTION : 8
END_HOST_CHECK : 1
END-REQUEST-920-PROTOCOL-ENFORCEMENT : 8 
END-REQUEST-921-PROTOCOL-ATTACK : 8 
END-REQUEST-930-APPLICATION-ATTACK-LFI : 8 
END-REQUEST-931-APPLICATION-ATTACK-RFI : 8 
END-REQUEST-932-APPLICATION-ATTACK-RCE : 8 
END-REQUEST-933-APPLICATION-ATTACK-PHP : 8 
END_XSS_CHECKS : 0 
END-REQUEST-941-APPLICATION-ATTACK-XSS : 8 
END-REQUEST-942-APPLICATION-ATTACK-SQLI : 8 
END-REQUEST-943-APPLICATION-ATTACK-SESSION-FIXATION : 8 
BEGIN_REQUEST_BLOCKING_EVAL : 1 
END-REQUEST-949-BLOCKING-EVALUATION : 8 
END-RESPONSE-950-DATA-LEAKAGES : 8 
END-RESPONSE-951-DATA-LEAKAGES-SQL : 8 
END-RESPONSE-952-DATA-LEAKAGES-JAVA : 8 
END-RESPONSE-953-DATA-LEAKAGES-PHP : 8 
END-RESPONSE-954-DATA-LEAKAGES-IIS : 8 
END-RESPONSE-959-BLOCKING-EVALUATION : 8 
END_CORRELATION : 3 
END-RESPONSE-980-CORRELATION : 8
```

So I think only END_XSS_CHECKS is really not need anymore. But maybe it would make sense if all the other markers are really needed or if we could express the functionality in an easier way.

**Comment by umarfarook882** [comment]
@dune73 yes. Except `END_XSS_CHECKS`, every other SecMarker are actively used in  `skipAfter`.

**Comment by dune73** [comment]
Addressed in PR #822.

**Comment by dune73** [comment]
Closing this in favor of PR #882.
